### PR TITLE
[RPA-865-2] Force validation of Target input for browser actions

### PR DESCRIPTION
**Pull request by marcin-depcik**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

---

I agree to the terms of the RunBotics Contributor License Agreement.

### CODE REVIEW

File: runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionFormPanel/widgets/BrowserTargetWidget/BrowserTargetWidget.tsx [hunk 1]
```diff
@@ -52,11 +52,14 @@ const BrowserTargetWidget: FC<WidgetProps> = (props) => {
...
         }
     }, []);
 
     const onChangeWrapper = (prefix: string, value: string) => {
-        if(prefix === 'None') {
+        if (!prefix || !value) {
+            props.onChange(null);
+        } else if (prefix === 'None') {
             props.onChange(value);
         } else{
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionFormPanel/widgets/BrowserTargetWidget/BrowserTargetWidget.tsx]: missing space after else

File: runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionFormPanel/widgets/BrowserTargetWidget/BrowserTargetWidget.tsx [hunk 2]
```diff
@@ -52,11 +52,14 @@ const BrowserTargetWidget: FC<WidgetProps> = (props) => {
...
             props.onChange(prefix + value);
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionFormPanel/widgets/BrowserTargetWidget/BrowserTargetWidget.tsx]: what if value is "sth" and prefix is false/null? this way -> propsOnChange(null)

Comment (marcin-depcik) [reply runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionFormPanel/widgets/BrowserTargetWidget/BrowserTargetWidget.tsx]: We want that when either prefix or value is falsy both inputs are invalid, means locked till user will set proper values. Method props.onChange(null) will force this behavior. Moreover, onMounted, prefix always gets value from PREFIX_OPTIONS, so I think scenario that prefix is falsy will never happen.

Comment (star-in-the-jar) [reply runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionFormPanel/widgets/BrowserTargetWidget/BrowserTargetWidget.tsx]: LeonardoDicaprioSoldGifGIF

**Review by star-in-the-jar — DISMISSED** [review]

**Review by star-in-the-jar — APPROVED** [review]

**Pull request merged**